server/lib/icon.coffee

```diff
@@ -6,6 +6,7 @@ log = require('printit')
 module.exports = icons = {}
 market = require './market'
 
+APPS_FOLDER = '/' + path.join 'usr', 'local', 'cozy', 'apps'
```

Comment:
It's the right way if you want the path to be portable to OS other than UNIX. And `/usr/local` is already a UNIX thing, so you won't be more portable by using `path.join`.